router/batchrouter/asyncdestinationmanager/klaviyobulkupload/klaviyobulkupload.go

```diff
@@ -397,7 +397,6 @@ func (kbu *KlaviyoBulkUploader) Upload(asyncDestStruct *common.AsyncDestinationS
 
 	return common.AsyncUploadOutput{
 		ImportingParameters: importParameters,
-		SucceededJobIDs:     successJobs,
```

Comment:
What about above code, do we need to compute successJobs?

Reply:
Yes, we are using them in `ImportingJobIDs` as a difference of all jobs and jobs which failed during upload itself.
`return common.AsyncUploadOutput{
		ImportingParameters: importParameters,
		FailedJobIDs:        failedJobs,
		FailedCount:         len(failedJobs),
		ImportingJobIDs:     successJobs,
		SuccessResponse:     DelimitedUploadRespErr,
		DestinationID:       destination.ID,
}`